polus-notebook-plugin/VERSION

```diff
@@ -1 +1 @@
-0.3.2
\ No newline at end of file
+0.3.8
```

Comment:
Since we are adding an input rather than fixing bugs, this should be a minor version change.

Please change version to `0.4.0` here and everywhere else in the code.

Reply:
Done.